templates/repo/issue/view_content/sidebar.tmpl

```diff
@@ -661,7 +661,7 @@
 						{{end}}
 
 						<div class="text right actions">
-							<div class="ui cancel button">{{.locale.Tr "settings.cancel"}}</div>
+							<button class="ui cancel button">{{.locale.Tr "settings.cancel"}}</button>
```

Comment:
I came here for asking why this change left the cancel button acting as submit button because I couldn't find an explanation debugging, but I saw #23337 and I am afraid I will need to revert it or wait to @delvh's pr being merged.